docs/content/getting-started.md

```diff
@@ -88,9 +88,14 @@ AWS_CREDS="$HOME/.aws/credentials"
 hypershift install \
   --oidc-storage-provider-s3-bucket-name $BUCKET_NAME \
   --oidc-storage-provider-s3-credentials $AWS_CREDS \
-  --oidc-storage-provider-s3-region $REGION
+  --oidc-storage-provider-s3-region $REGION \
+  --enable-defaulting-webhook true
```

Comment:
can you specify you can either use webhook or specify you release image/stream below in the create command?

Reply:
Resolved thru comment on L129